cdnsystem/server/server.go

```diff
@@ -116,6 +121,15 @@ func New(cfg *config.Config) (*Server, error) {
 	return s, nil
 }
 
+func (s *Server) startDevFileServer() (err error) {
+	if !s.config.Verbose {
+		return
+	}
+	p, _ := filepath.Abs(s.storage.GetHomePath())
+	http.Handle("/", http.FileServer(http.Dir(p)))
+	return http.ListenAndServe(fmt.Sprint(":", s.config.DownloadPort), nil)
```

Comment:
Same port with nginx ?

Reply:
yes, when use local file server ,nginx is redundancy